webapp/src/components/App/Singlestudy/HomeView/InformationView/CreateVariantModal/index.tsx

```diff
@@ -51,54 +60,57 @@ function CreateVariantModal(props: Props) {
     }
   };
 
+  ////////////////////////////////////////////////////////////////
+  // Hooks
+  ////////////////////////////////////////////////////////////////
+
   useEffect(() => {
-    setVersionSourceList(createListFromTree(tree));
+    setSourceList(createListFromTree(tree));
   }, [tree]);
 
+  ////////////////////////////////////////////////////////////////
+  // JSX
+  ////////////////////////////////////////////////////////////////
+
   return (
-    <BasicDialog
-      open={open}
-      onClose={onClose}
+    <FormDialog
       title={t("studies.createNewStudy")}
-      actions={
-        <>
-          <Button variant="text" color="primary" onClick={onClose}>
-            {t("global.cancel")}
-          </Button>
-          <Button
-            sx={{ mx: 2 }}
-            color="primary"
-            variant="contained"
-            onClick={onSave}
-          >
-            {t("global.create")}
-          </Button>
-        </>
-      }
+      titleIcon={AddCircleIcon}
+      open={open}
+      onCancel={onClose}
+      onSubmit={handleSubmit}
+      config={{
+        defaultValues: {
+          name: "",
+          sourceId: "",
```

Comment:
set the default sourceId to parentId (well you should add back the parentId props then)